src/app/attendances/page.tsx

```diff
@@ -0,0 +1,14 @@
+import TopNavBar from '../molecules/top-navigation-bar';
+import AttendanceStatus from '../molecules/attendance-status';
+import SubjectAttend from '../organisms/subject-attend';
```

Comment:
```suggestion
import TopNavBar from "@/molecules/top-navigation-bar";
import SubjectAttend from "@/organisms/subject-attend";
```
ネストが 1 階層深くなっていてこの相対パスだとそれらのモジュールにたどり着けません. `../../` か `@/` で始めることで, `src/` 配下のパスにできます.